addons/website/static/tests/core/interaction.test.js

```diff
@@ -1206,6 +1206,31 @@ describe("dynamic attributes", () => {
         expect("span").toHaveClass("b c");
     });
 
+    test("t-att-class does not touch unrelated classes on destroy", async () => {
+        class Test extends Interaction {
+            static selector = "span";
+            dynamicContent = {
+                "_root:t-att-class": () => ({ b: true }),
+            };
+        }
+
+        const { core, el } = await startInteraction(
```

Comment:
General comment for this test : please look at the other tests for the template and the expects.
With these the tests are more reliable since adding an extra space in the template in the `toBe` might cause an error for nothing

Reply:
Adapted - except for the test about CSS property's priority - because the tools are too broken.